packages/block-editor/src/store/selectors.js

```diff
@@ -225,18 +225,21 @@ export const __unstableGetClientIdsTree = createSelector(
  *
  * @return {Array} ids of descendants.
  */
-export const getClientIdsOfDescendants = ( state, clientIds ) => {
-	const collectedIds = [];
-	for ( const givenId of clientIds ) {
-		for ( const descendantId of getBlockOrder( state, givenId ) ) {
-			collectedIds.push(
-				descendantId,
-				...getClientIdsOfDescendants( state, [ descendantId ] )
-			);
+export const getClientIdsOfDescendants = createSelector(
```

Comment:
This should have been memoized anyway, I don't think that's what caused the regression though but let's see.